README.md

```diff
@@ -836,6 +836,12 @@ ha_cluster_constraints_ticket:
 You may take a look at
 [an example](#creating-a-cluster-with-resource-constraints).
 
+#### `ha_cluster_manage_firewall`
```

Comment:
In mssql, I use `mssql_firewall_configure` variable for this purpose. That's because historically, I was using the same format for `mssql_ha_configure`. Do we want to make this consistent across all roles?

Reply:
I would put this variable description to the top, perhaps right bellow `ha_cluster_enable_repos`. That's where system-like settings are described. Here, at the bottom, cluster configuration is documented.

Reply:
Thanks a lot, @tomjelinek! I've pushed another commit to fix it. (I will or please do squash before merging this pr.)